File: telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/service/RandomPhraseService.java
package com.github.bumblebee.service;

import org.springframework.stereotype.Service;

import java.util.Random;

@Service
public class RandomPhraseService {

    private static final String[] SURPRISE_PHRASES = {"Wut?", "Whaaat?"};
    private static final String[] NO_PHRASES = {"No", "Mmmm... No", "NO WAY", "Not today"};

    private Random random = new Random();

    public String surprise() {
        return SURPRISE_PHRASES[random.nextInt(SURPRISE_PHRASES.length)];
    }

    public String no() {
        return NO_PHRASES[random.nextInt(NO_PHRASES.length)];
    }
}

File: telegram-bot-bumblebee-core/src/main/kotlin/com/github/bumblebee/service/RandomPhraseService.kt
package com.github.bumblebee.service

import org.springframework.stereotype.Service
import java.util.*

@Service
class RandomPhraseService {

    private val random = Random()

    fun surprise(): String {
        return SURPRISE_PHRASES[random.nextInt(SURPRISE_PHRASES.size)]
    }

    fun no(): String {
        return NO_PHRASES[random.nextInt(NO_PHRASES.size)]
    }

    companion object {
        private val SURPRISE_PHRASES = arrayOf("Wut?", "Whaaat?")
        private val NO_PHRASES = arrayOf("No", "Mmmm... No", "NO WAY", "Not today")
    }
}
